_config/videos.yml

```diff
@@ -39,6 +53,7 @@
   date: 2020-04-02
   tags:
     - quarkus
+      - school timetabling
```

Comment:
Wrong indentation. Alters the `quarkus` on the line above making the video disappear from related videos on the Quarkus compatibility page.

Note that even if you fix the tag, the video is not added to related videos on School Timetabling use case because use case layout doesn't have "related videos" section.

Reply:
Fixed identation, should be good now :)

Yes, the "related videos" isn't used yet for use case layout. It's an open issue.